src/patternfly/base/_globals.scss

```diff
@@ -137,5 +137,5 @@
 @import "./themes/dark/globals";
 
 @include pf-theme-dark($pf-theme-dark-prefix: ":root") {
-  @include globals;
+  @include pf-_globals;
```

Comment:
I don't love this syntax. Is there a way around having `-_`?

Reply:
Agree, updated it to have a `pf-theme-dark` prefix and drop the `_` in the mixin name.